docs/integrations/apps/guide/types.mdx

```diff
@@ -22,7 +22,7 @@ Once granted, the app can request a permanent `access_token` for making REST API
 Single-click apps can use [App Extensions](/docs/integrations/apps/app-extensions).
 
 ## Connector
-Connector apps use manual OAuth token creation instead of the single-click app flow. Store owners manually generate [store-level API credentials](/docs/start/authentication/api-accounts#revoking-store-level-api-credentials) and enter them into the app's configuration. While single-click apps are recommended, the following use cases might not be compatible:
+Connector apps use manual OAuth token creation instead of the single-click app flow. Store owners and authorized users manually generate [store-level API credentials](/docs/start/authentication/api-accounts#revoking-store-level-api-credentials) and enter them into the app's configuration. While single-click apps are recommended, the following use cases might not be compatible:
```

Comment:
removed passive voice
```suggestion
Connector apps use manual OAuth token creation instead of the single-click app flow. Store owners and authorized users manually generate [store-level API credentials](/docs/start/authentication/api-accounts#revoking-store-level-api-credentials) and enter them into the app's configuration. While we recommend single-click apps, the following use cases might not be compatible:
```